### DISCUSSION TITLE
How to navigate to previous screen without going to portrait mode?

### DISCUSSION THREAD

**Discussion by pavankumar-clasc** [jhomlala/betterplayer#811] [Q&A]
I have put the video player to open in full-screen mode by putting the fullScreenByDeafult parameter in BetterPlayerControlsConfiguration to true. But When I clicked the back button on full-screen mode It is going to portrait video mode. I want to eliminate that portrait screen and to add like when we hit the back button It will navigate to the previous page. Could you please help me out to implement that?

**Comment by tanishqarora97** [comment]
Any leads on this?

**Comment by sruti-sunyam** [comment]
add an eventlistener to the player then track the event listner which is called when you are clicking on the back button then at that onlick route it to the page you want to go back to
